SecretSanta/src/SecretSanta.Api/Dto/UpdateUser.cs

```diff
@@ -0,0 +1,7 @@
+namespace SecretSanta.Api.Dto
+{
+    public class UpdateUser{

```

Comment:
Nit: Typically we put braces on their own line
```suggestion
    public class UpdateUser
    {

```